rhel/SOURCES/clickhouse.service

```diff
@@ -1,20 +0,0 @@
-[Unit]
```

Comment:
Don't we still need `clickhouse.service` for the AMI/OVF distributions?

Reply:
We use supervisor for starting Clickhouse in AMI/OVF. The same as in Docker container.

Reply:
👍